SUPPORTED-FORMATS.md

```diff
@@ -445,6 +504,20 @@ If your tool is supported, but some properties are missing (icon, URL, etc.), pl
                 -
             </td>
         </tr>
+        <tr>
+            <td>
+                cpplint
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                Cpplint
+            </td>
+            <td>
+                -
+            </td>
+        </tr>
```

Comment:
You might not have noticed line 1:
```
<!--- DO NOT EDIT - Generated by ParserRegistry at 2021-11-24T08:25:06.342217-->
```
You need to provide the description in the class.